kqueen/config/default_policy.json

```diff
@@ -0,0 +1,22 @@
+{
+    "cluster:create": ["admin"],
+    "cluster:delete": ["admin"],
+    "cluster:get": "all",
+    "cluster:list": "all",
+    "cluster:update": "all",
+    "organization:create": ["superadmin"],
+    "organization:delete": ["superadmin"],
+    "organization:get": "all",
+    "organization:list": "all",
```

Comment:
Maybe `organizatoin:list` just superadmin ... do we need it somewhere else? Is this capability required for duplicates in namespaces?

Reply:
I think it is, but when I do out of scope API calls from UI, I use service account, which has superadmin privileges anyways, so it should be possible to restrict organization:list to superadmin only. I'll try it tomorrow.